content/en/integrations/guide/aws-organizations-setup.md

```diff
@@ -0,0 +1,131 @@
+2. **An account administrator has enabled Trusted Access with AWS Organizations**: Refer to [AWS Documentation][3] on how to enable trusted access between StackSets and Organizations to create & deploy stacks using service-managed permissions.
+
+## Setup
+
+To get started, go to the [AWS Integration configuration page][1] in Datadog and click on **Add AWS Account(s)** -> **Add Multiple AWS Accounts** -> **Cloudformation StackSet**.
+
+On the Datadog console, click **Launch CloudFormation StackSet**. This opens the AWS Console and loads a new CloudFormation StackSet. Keep the default choice of “Service-managed permissions” on AWS.  
+  
+Follow the steps below on the AWS console to create and deploy your stackset:
+
+1. **Choose a Template**  
+Copy the Template URL from the Datadog console to use it in the `Specify Template` parameter in the StackSet.
+
+
+2. **Specify StackSet details**
+    - Select your Datadog API key on the Datadog console and use it in the `DatadogApiKey` parameter in the StackSet.
+    - Select your Datadog APP key on the Datadog console and use it in the `DatadogAppKey` parameter in the StackSet.
+
+    - *Optionally:*
+        - Enable [Cloud Security Posture Management][5] (CSPM) to scan your cloud environment, hosts, and containers for misconfigurations and security risks.
+        - Disable metric collection if you do not want to monitor your AWS infrastructure. (recommended only for [Cloud Cost Management][6] (CCM) or [Cloud Security Posture Management][5] (CSPM) only use cases)
+
+
+3. **Configure StackSet options**  
+Use the default option of "Execution configuration" to be set to inactive so the StackSet performs one operation at a time.
+
+4. **Set deployment options**
+    - You can set your “Deployment targets” to either deploy the Datadog integration across an Organization or one or more Organizational Units.
+
+
+    - Keep “Automatic deployment” enabled in order to automatically deploy the Datadog AWS Integration in new accounts that are added to the Organization or OU.
+
+    - Under “Specify regions”, select a region in which you’d like to deploy the integration.   
+      **NOTE**: The StackSet will set up global IAM resources that are not region specific. If multiple regions are selected in this step, the deploy will fail. 
+
+
+    - Use the default settings under “Deployment options” to be set to Sequential so StackSets operations are deployed into one region at a time.
+
+5. **Review**  
+    Move to the Review page and Click Submit. This launches the creation process for the Datadog StackSet. This could take a while depending on how many accounts need to be integrated. Ensure that the StackSet successfully creates all resources before proceeding.
+
+    After the stacks are created, go back to the AWS integration config page in Datadog and click **Done**. Please wait for a few minutes to see metrics & events reporting from your newly integrated AWS accounts.
+
+
```

Comment:
Might be doing one quick call out on how to uninstall the AWS Integration on all accounts or roll this back. Could maybe just reference the AWS documentation for uninstalling a stack set